src/giskardpy/utils.py

```diff
@@ -514,4 +515,78 @@ def add_edges(root, root_dot_name, visibility_level):
                     add_edges(c, proposed_dot_name, visibility_level)
 
     add_edges(root, root.name, visibility_level)
-    return graph
\ No newline at end of file
+    return graph
+
+
+
+def rospkg_exits(name):
+    """
+    checks whether a ros package with the given name and version exits
+    :param name: the name and version of the ros package in requirements format e.g. giskard_msgs<=0.1.0
+    :return: True if it exits else False
+    """
+    r = rospkg.RosPack()
+    try:
+        l = name.split('=')
+        if l[0].endswith('<'):
+            m = r.get_manifest(l[0][:-1])
+            version_m = m.version.split('.')
+            version_d = l[len(l) - 1].split('.')
+            k = max(len(version_m), len(version_d))
+            for i in range(k):
+                if version_m[i] < version_d[i]:
+                    return True
+                elif version_m[i] > version_d[i]:
+                    print("found ROS package " + l[0][:-1] + "==" + str(
+                        m.version) + " but " + name + " is required")
```

Comment:
Please always use single quotes for strings, so that it's consistent with the rest of my code. And use '{}'.format(), to build strings.